pages/dashboards/views/slu_ww.py

```diff
@@ -0,0 +1,155 @@
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import render
+from django.utils.text import slugify
+from django.views import View
+
+from ..models import DashboardData
+from ..visualisation.slu_ww import (
+    get_all_sites_plot,
+    get_compiled_data,
+    get_quant_overview_plot,
+    get_single_site_plot,
+)
+from ..visualisation.utils import plot_html_from_json
+
+
+class SluWasterWater(View):
```

Comment:
```suggestion
class SluWasteWater(View):
```

typo :)

Reply:
Gahh!! I am spoiled by VS code auto addition and made the same typo in multiple files 😆 

Fixed it now, thank you for spotting.